src/app/app.module.ts

```diff
@@ -16,7 +17,8 @@ import { AdminProductsListComponent } from './views/admin-views/admin-products-l
   imports: [
     BrowserModule,
     AppRoutingModule,
-    SharedModule
+    SharedModule,
+    NgbModule,NgbAlertModule,
```

Comment:
Remove this as we have already in shared module

Reply:
Resolved.

Reply:
Not resolved please check again

Reply:
Resolved.